src/tests/ll_verification.py

```diff
@@ -4019,7 +4019,12 @@ def ll_con_mas_bv_13_c(transport, upperTester, lowerTester, trace):
         hasFeatures, expectedFeatures = readLocalFeatures(transport, lowerTester, trace)
         hasFeatures, upperFeatures    = readLocalFeatures(transport, upperTester, trace)
         upperLocalFeatures = toNumber(upperFeatures)
-        expectedMaskedFeatures = toNumber(expectedFeatures) & (upperLocalFeatures | ~LL_FEAT_BIT_MASK_VALID)
+        """
+            Keep octets 1-7, and do the logical and on octet 0; also ignore the non-valid bits
+            See BT Core spec V5.2, Vol 6. Part B chapter 4.6
+        """
+        expectedMaskedFeatures = toNumber([(upperFeatures[0] & 0xFF) & expectedFeatures[0]] + list(expectedFeatures[1:7]))
```

Comment:
Remove "& 0xFF" as upperFeatures[0] is already one byte.

Reply:
Done